napalm/eos/pyeapi_syntax_wrapper.py

```diff
@@ -32,9 +32,11 @@ def run_commands(self, commands, **kwargs):
         :param kwargs: other args
         :return: list of outputs
         """
-        if isinstance(commands, str):
-            new_commands = [cli_convert(commands, self.cli_version)]
-        else:
-            new_commands = [cli_convert(cmd, self.cli_version) for cmd in commands]
+        fn0039_transform = kwargs.pop("fn0039_transform", True)
```

Comment:
Shouldn't this default to False if `fn0039_transform` doesn't exist?

Reply:
Oh, I thought I'd default this to True as we may want to disable this behaviour only for the config-related methods (the getters would always convert the commands under the hood)... so it was mostly a matter of whether we update every getter (everywhere `run_commands` is called) or just `_load_config`. Not sure if I'm explaining this well?

Reply:
Okay, that makes sense.

So internal to NAPALM for show commands, we would do this automatic conversion. But then for any configuration changes by end users, they would default to `no conversion` (with an option they could specify that would do the conversion).

Is that the correct understanding?

Reply:
Yes, correct, much better worded like this. :smile: